requirements-devel.txt

```diff
@@ -21,12 +21,3 @@
 # In applying this license, CERN does not
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
-#
-# TODO: Add development versions of some important dependencies here to get a
-#       warning when there are breaking upstream changes, e.g.:
-#
-#     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
-#     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
-
--e git+git://github.com/inveniosoftware/invenio-userprofiles.git#egg=invenio-userprofiles
```

Comment:
Why do we need to remove the dependency on `invenio-userprofiles`? It is one of the modules which is used by `invenio-accounts-rest`. Same question for invenio-rest.